src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -107,8 +112,8 @@ namespace ts.codefix {
         },
     });
 
-    const enum InfoKind { TypeLikeDeclaration, Enum, Function, ObjectLiteral, JsxAttributes }
-    type Info = TypeLikeDeclarationInfo | EnumInfo | FunctionInfo | ObjectLiteralInfo | JsxAttributesInfo;
+    const enum InfoKind { TypeLikeDeclaration, Enum, Function, ObjectLiteral, JsxAttributes, Signature }

```

Comment:
this might not be the place to fix it, but 

1. why is this a const enum instead of a normal one? Performance shouldn't be a concern here.
2. I think this should be formatted one entry per line, unless there's a reason not to.
3. It would be *little* easier to read if the enum were declared earlier, before its first use in the file. But I'm not sure about that.

Reply:
Ok, I've changed the formatting.